src/hardware/hantek-dso/protocol.h

```diff
@@ -131,6 +140,7 @@ enum capturestates {
 	CAPTURE_FILLING = 1,
 	CAPTURE_READY_8BIT = 2,
 	CAPTURE_READY_9BIT = 7,
+	CAPTURE_READY2250 = 3,
```

Comment:
are these meant to bit bit masks or what?  Either way, it seems that it should be placed  betwen 8 and 9 bit